wire-compiler/src/main/java/com/squareup/wire/schema/WireRun.kt

```diff
@@ -271,6 +273,8 @@ data class WireRun(
           module = module,
           profileLoader = schemaLoader,
         )
+        // Ensure the out directory is created.
+        fs.createDirectories(outDirectory)
```

Comment:
This is a bit of a strange change but it seems like there exists a test which ensures this behavior: https://github.com/square/wire/blob/cad71b5754551cf541386b73c88c51e97f4dea75/wire-compiler/src/test/java/com/squareup/wire/schema/WireRunTest.kt#L1004-L1023